src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -33,6 +33,11 @@ public static object GetUninitializedObject(Type type)
             return FormatterServices.GetUninitializedObject(type);
         }
 
+        #if FEATURE_STARTUP_MINOPT
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        public static extern void SetOptimizationLevel(int level);
```

Comment:
Is that gap because of you are snapped to last released version (e.g. .NET Core 2.1), or because of you have a lot of changes that are not upstream?

If it is the first case, it would help to compare with what you are snapped to.